anomaly_detector/exception/exceptions.py

```diff
@@ -1,9 +1,17 @@
 """Fact Store Exception - When required env vars are not set we throw these exceptions."""
 
 
-class factStoreEnvVarNotSetException(Exception):
+class FactStoreOffline(Exception):
     """Fact Store env var validator."""
 
     def __init__(self, msg="fact store url env var not set"):
         """Initialize message."""
         self.message = msg
+
+
+class EmptyDataSetNotAllowed(ValueError):
+    """Empty Dataset validator."""
+
+    def __init__(self, msg="Reading from empty file or dataset streaming has no data."):
+        """Initialize message."""
+        self.message = msg
```

Comment:
We should have our system errors provide meaning full message. Without this you will see the program crash but not know why.